cluster/src/test/java/com/hartwig/pipeline/smoke/SmokeTest.java

```diff
@@ -27,9 +55,72 @@ public void runFullPipelineAndCheckFinalStatus() {
                 .setId("CPCT12345678")
                 .mode(Arguments.Mode.FULL)
                 .runId("smoke-" + noDots(version))
-                .runGermlineCaller(false)
-                .build());
+                .runGermlineCaller(false).sbpApiRunId(RUN_ID)
+                .sbpApiUrl(apiUrl).rclonePath(RCLONE_PATH).rcloneS3RemoteDownload(S3_REMOTE).rcloneS3RemoteUpload(S3_REMOTE)
+                .sbpS3Url("s3.us-east-2.amazonaws.com").rcloneGcpRemote(GCP_REMOTE)
+                .upload(true).build();
+
+        String destinationBucket = bucketName(arguments);
+        String setName = setName(arguments);
+
+        PipelineState state = victim.start(arguments);
         assertThat(state.status()).isEqualTo(PipelineStatus.QC_FAILED);
+
+        List<String> rcloneListing = listRemoteFilesNamesOnly(format("%s:%s/%s/", S3_REMOTE, destinationBucket, setName));
```

Comment:
This is a little hairy, maybe good enough to just break out into smaller methods for readability. Or another class (ManifestAssertion?) if you have a nice idea